src/test/ui/issues/issue-6596-1.rs

```diff
@@ -1,7 +1,7 @@
 macro_rules! e {
     ($inp:ident) => (
         $nonexistent
-        //~^ ERROR unknown macro variable `nonexistent`
+        //~^ ERROR expected expression, found `$`
```

Comment:
Not all these diagnostics go through the generic expected-found infra, unfortunately, so those who do not will report something like this.
These cases need to be migrated to `expect_one_of` or something similar, but I didn't do that.

Reply:
Or the old "unknown macro variable" can be put at the end of bump.
Now when it separated from normalization it shouldn't mess up anything.

Ideally, unknown `$var` should be a "fatal" error propagated upwards, but `bump` doesn't currently return `Result` and changing that would be pretty invasive.